spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -1,26 +1,70 @@
 describe 'orifice-calculator-viewmodel-spec', ->
   viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
 
+  beforeEach ->
+    viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
+
+  itBehavesLikeMandatoryField = (field) ->
+    it 'has mandatory field validation', ->
+      field('')
+      expect(field.isValid()).toBeFalsy()
+
+  itBehavesLikeIntegerField = (field) ->
+    it 'should only accept integer', ->
+      field(123)
+      expect(field.isValid()).toBeTruthy()
```

Comment:
Similar logic applies on toBeTruthy

provided that the value is not any of the falsy list it will pass.
So please check against true or false